src/software/unix_full_system_main.cpp

```diff
@@ -61,15 +62,20 @@ int main(int argc, char** argv)
         {
             TracySetProgramName("Thunderbots: Yellow");
         }
-
-        LoggerSingleton::initializeLogger(args.runtime_dir);
+        auto proto_logger = std::make_shared<ProtoLogger>(args.runtime_dir, []() {
+            return std::chrono::duration<double>(
+                       std::chrono::system_clock::now().time_since_epoch())
+                .count();
+        });
+        LoggerSingleton::initializeLogger(args.runtime_dir, proto_logger);
```

Comment:
One instance of the new `ProtoLogger` is created at the start of fullsystem and dependency injected into the logger and the other unix senders and receivers.